sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - `begin_recognize_receipts` and `begin_recognize_receipts_from_url` now return `RecognizedForm`.
 - `requested_on` renamed to `training_started_on` and `completed_on` renamed to `training_completed_on` on `CustomFormModel`
 and `CustomFormModelInfo`
+- `FieldText` has been renamed to `FieldData`
+- `FormContent` has been renamed to `FormElement`
+- Parameter `include_text_content` has been renamed to `include_field_elements`
```

Comment:
nit: Might want to include which methods `include_text_content` is a part of and where `text_content` appears